source/Octodiff/Octodiff.csproj

```diff
@@ -0,0 +1,49 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net40</TargetFrameworks>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <AssemblyName>Octodiff</AssemblyName>
+    <PackageId>Octodiff</PackageId>
+    <OutputType>Exe</OutputType>
```

Comment:
This should be moved into the `PropertyGroup` on line 23.

It looks like we always used to produce an exe, then when we did the netcore port we decided that build was only going to be an library (for compatiblity). So we want an exe output for the `net40` build, otherwise a `netstandard` `dll`.